examples/cluster/plot_cluster_comparison.py

```diff
@@ -42,21 +42,34 @@
 blobs = datasets.make_blobs(n_samples=n_samples, random_state=8)
 no_structure = np.random.rand(n_samples, 2), None
 
+# Anisotropicly distributed data
+random_state = 170
+X, y = datasets.make_blobs(n_samples=n_samples, random_state=random_state)
+transformation = [[0.6, -0.6], [-0.4, 0.8]]
+X_aniso = np.dot(X, transformation)
+aniso = (X_aniso,y)
+
+# Different variance
+varied = datasets.make_blobs(n_samples=n_samples,
+                                cluster_std=[1.0, 2.5, 0.5],
+                                random_state=random_state)
+
+
 colors = np.array([x for x in 'bgrcmykbgrcmykbgrcmykbgrcmyk'])
 colors = np.hstack([colors] * 20)
 
 clustering_names = [
     'MiniBatchKMeans', 'AffinityPropagation', 'MeanShift',
     'SpectralClustering', 'Ward', 'AgglomerativeClustering',
-    'DBSCAN', 'Birch']
+    'DBSCAN', 'Birch', 'GaussianMixture']
 
-plt.figure(figsize=(len(clustering_names) * 2 + 3, 9.5))
+plt.figure(figsize=(len(clustering_names) * 2 + 3, 12.5))
 plt.subplots_adjust(left=.02, right=.98, bottom=.001, top=.96, wspace=.05,
                     hspace=.01)
 
 plot_num = 1
-
-datasets = [noisy_circles, noisy_moons, blobs, no_structure]
+# noisy_circles, noisy_moons, blobs, no_structure, varied
+datasets = [noisy_circles, noisy_moons, blobs, no_structure, varied,aniso]
```

Comment:
It's not a big deal but I think it's better to put no_structure at the end.
It's just to easily compare the result between the gaussian datas.